projects/schematics/src/shared/utils/dependency-utils_spec.ts

```diff
@@ -106,7 +112,13 @@ describe('dependency-util', () => {
         SPARTACUS_CDC,
       ]);
 
-      expect(result).toEqual([SPARTACUS_USER, SPARTACUS_ASM, SPARTACUS_CDC]);
+      // TODO: Double-check this.
```

Comment:
Why are schematics unit tests updated in the scope of this PR?

Reply:
I added this TODO. I believe the reason for this is because the ASM module now depends on the Cart module, therefore this list of dependencies needs to be updated. The TODO is to check with a senior SPA dev.

Reply:
@justinlee01 Can you explain this?

Reply:
@plabadie I am awaiting your comment.